### ISSUE TITLE
Command Bar: The aria allowed attributes aria-orientation-"horizontal" is failing

### ISSUE THREAD

**Issue by qazaahir** [OfficeDev/office-ui-fabric-react#11288]
The aria allowed attributes aria-orientation-"horizontal" is failing. We don't have access to this variable and cannot make the changes

To Reproduce
If you run

https://developer.microsoft.com/en-us/fabric#/controls/web/detailslist/variablerowheights

in code pen you will observe the bug
Screenshots
![image](https://user-images.githubusercontent.com/57369169/69463476-83db4c80-0d30-11ea-8d8f-be6ae3c18fe2.png)
![image](https://user-images.githubusercontent.com/57369169/69463484-89d12d80-0d30-11ea-96b8-c0c19a88f871.png)

**Comment by aneeshack4** [comment]
@qazaahir Thanks for reaching out to us! This div has `role="presentation"`. Based off the [ARIA documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-orientation_attribute), I think this failure is because this ARIA property can only be used with certain ARIA roles and presentation isn't on of them. The parent div has `role="menubar"` which is an ARIA role that's compatible with `aria-orientation` because [it inherits.](https://www.w3.org/TR/wai-aria/#aria-orientation) so maybe we can just remove the `role="presentation"` from this child div? Looping in component owners @micahgodbolt @joschect for their thoughts.

**Comment by AbdelrhmanMagdy** [comment]
@aneeshack4  The attributes of the div that contain the role="presentation" aren't exposed from the command bar component .. it's only implemented throw items prop .. is there a clean way to access and change the role value inside this div?

**Comment by micahgodbolt** [comment]
this is a dupe of #11039 and has just recently been solved by #11224

**Comment by aneeshack4** [comment]
@AbdelrhmanMagdy looks like Micah's PR solves this issue - are you sure you're using the latest release of Fabric so you get the fix?

**Comment by AbdelrhmanMagdy** [comment]
> @AbdelrhmanMagdy looks like Micah's PR solves this issue - are you sure you're using the latest release of Fabric so you get the fix?

yes it's fixed in the latest version .. thank you
